Move all assets to new folder

nmuesch opened the pull request:
### What does this PR do?

Moves all assets (except for metadata.csv) into a new assets folder
Also adds a new assets key to the manifest.json file for each integration to point to the relative path of these new assets. 
Updates the ddev validations to accept this new folder structure (See this commit for those changes - https://github.com/DataDog/integrations-core/pull/3768/commits/3ff033074de9255f07dbb965046842f636c7ac98)
Updates the ddev create templates to use this new assets folder

### Motivation

After internal discussions we're standardizing the way we add and read new data from this repo. 

### Additional Notes

I made a script here that did this work:

```
from datadog_checks.dev.tooling.utils import get_valid_integrations, load_manifest, get_manifest_file
from datadog_checks.dev.tooling.constants import set_root

import os
import json

ROOT = '/Users/nicholasmuesch/git/integrations-core/'
NEW_MANIFEST = {
    'assets': {
        'monitors': {},
        'dashboards': {},
        'service_checks': '{}'
    }
}

def get_updated_manifest(service_check_path):
    NEW_MANIFEST['assets']['service_checks'] = service_check_path
    return NEW_MANIFEST


if __name__ == "__main__":
    set_root(ROOT)
    checks = get_valid_integrations()
    for check in checks:
        print(f'Modifying {check}')
        current_dir = f'{ROOT}{check}'
        manifest_file = get_manifest_file(check)
        manifest = load_manifest(check)

        # Make required dirs
        os.mkdir(f'{current_dir}/assets')
        if os.path.exists(f'{current_dir}/logos'):
            os.mkdir(f'{current_dir}/assets/logos')
            os.rename(f'{current_dir}/logos/', f'{current_dir}/assets/logos/')

        # Move contents from root to new dirs
        os.rename(f'{current_dir}/service_checks.json', f'{current_dir}/assets/service_checks.json')

        # Update the manifest file to include this new locations
        manifest.update(get_updated_manifest(f'{check}/assets/service_checks.json'))

        with open(manifest_file, 'w+') as f:
            f.write(f'{json.dumps(manifest, indent=2)}\n')
```

### Review checklist (to be filled by reviewers)

- [ ] PR title must be written as a CHANGELOG entry [(see why)](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have `changelog/` and `integration/` labels attached
- [ ] Feature or bugfix must have tests
- [ ] Git history [must be clean](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] If PR adds a configuration option, it must be added to the configuration file.

l0k0ms commented:
Doc wise we checked with @nmuesch and it's good to go, just ping us before merging it so that we can update one variable in our build script.

nmuesch commented:
@gzussa I updated the ddev create templates to move the service checks file inside the assets folder and added the new assets section to the manifest.
